docs/fundamentals/jcloud/advanced.md

```diff
@@ -73,6 +73,101 @@ executors:
           type: efs
 ```
 
+### GPU support
+
+We have GPU support in JCloud. We have `2` kind of GPU usages, `shared` or `dedicated`.
+
+```{note}
+Due to enable GPU support may require to cluster to dynamically provision new GPU nodes. It may take extra 2-3 mins till all flows are running in the cluster.
+```
```

Comment:
```suggestion
```{note}
When using GPU resources, it may take an extra 2-3 mins until all Flows are running in the cluster.
This is because enabling GPU support may require the cluster to dynamically provision new GPU nodes.
```
```